hrm-domain/scheduled-tasks/hrm-data-pull/pull-profiles.ts

```diff
@@ -178,7 +178,10 @@ export const pullProfiles = async (startDate: Date, endDate: Date) => {
       1) 'totalCount' property, which I think is wrong, so I'm deleting it
     */
       delete (profile as any).totalCount;
-      const date = format(parseISO(profile.updatedAt.toISOString()), 'yyyy/MM/dd');
+      const date = format(
+        parseISO(profile.updatedAt.toString() ?? profile.createdAt.toString()),
```

Comment:
Thanks @mythilytm for figuring this out! @stephenhand another thing to mention here is that  typescript believes `profile.updatedAt` is a Date, but at runtime it's a string. Is it possible this points to a bigger issue in our data access layer?